src/Hydrodynamics/SphSimulation.cpp

```diff
@@ -1663,6 +1729,5 @@ void SphSimulation<ndim>::SmoothParticleQuantity
  (const int Npart,
   FLOAT *values)
 {
-
```

Comment:
Was once a dream that I didn't have time to actually fulfil.  Not all dreams come true...... Deleted!!